docs/blocks/javascript.md

```diff
@@ -1 +1,61 @@
 # JavaScript Code
+
+It allows to run JavaScript code in the browser.
```

Comment:
```suggestion
It allows to run JavaScript code in a separate  web worker. It is therefore ideal for performing heavy computations in the background.
```
I think it's important to distinguish for experienced js developer. Otherwise they will try to access DOM, which is not possible within web worker.

Reply:
I removed the "heavy" since all-in-all it should be within a couple of seconds at most to have a good user experience.